<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWindow</class>
 <widget class="QMainWindow" name="PlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(0, 0, 0);
}
* {
    background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>500</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    color: rgb(255, 255, 0);
	font-size: 120px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>PACMAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backpushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 0);
	font-size: 30px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hostButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 0);
	font-size: 35px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 0);
	font-size: 35px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
